### PR TITLE
Use Jekyll to generate site

### PR DESCRIPTION
**Pull request by cmfcmf**
This PR introduces Jekyll to generate the documentation site.
Jekyll is automatically used by GitHub pages - no additional build step is required.

- The page is now responsive and #2 should no longer be needed.
- Obviously the Padauk SDCC Tutorial page is still empty - but it seems like a good place to start.
- I did not link to any of the SDCC examples and header repositories, because their structure is not yet clear.
- I kept the old pinout diagrams of the PMS154 for now.

image

---

image

---

image

### CODE REVIEW

**Comment by cpldcpu** [comment]
Awesome! I guess I need to install jykell locally to check this out before deployment. Will look into this asap.

**Comment by cmfcmf** [comment]
> I guess I need to install jykell locally to check this out before deployment. Will look into this asap.

We could setup a service like netlify to automatically build previews of the site for pull requests. I've enabled it on my fork for testing purposes, and it appears to work fine. Here is an example pull request with a link to a preview site: https://github.com/cmfcmf/free-pdk.github.io/pull/1

image

https://deploy-preview-1--reverent-sinoussi-f065ff.netlify.app/

**Comment by cpldcpu** [comment]
That looks great, need to check it out. Btw - I created a development branch and changed this pull request to the new branch.

File: index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+*Free PDK* is an effort to create an open source alternative to the proprietary
+[Padauk](http://www.padauk.com.tw/index_en.aspx) µC programmer,
+as well as adding support to [SDCC](http://sdcc.sourceforge.net/) for Padauk µCs.
+The main focus is on supporting µCs of two Padauk series:
+- [M series OTP](http://www.padauk.com.tw/en/product/index.aspx?kind=41) (OTP = *one time programmable*)
+- [F series MTP](http://www.padauk.com.tw/en/product/index.aspx?kind=42) (MTP = *multiple time programmable*)
+
+Padauk µCs are extremely cheap (as cheap as $0.03 per µC), which is why
+David from the EEVblog made the [first video](https://youtu.be/VYhAGnsnO7w)
```
Comment (cpldcpu) [inline index.md:15]: He is actually named "Dave" :)

**Review by freepdk — APPROVED** [review]
This looks great. Keep going :-)

**Review by cpldcpu — COMMENTED** [review]
The header/footer on the page for the individual devices seems to be huge! There is only about a third of the page for content. Would it make sense to change this to fixed layout that always displays the full pinout information?

grafik

**Review by cpldcpu — APPROVED** [review]

**Comment by cpldcpu** [comment]
anyways - pushing this as it is definitely a huge leap over the current state. Can add more changes later.

**Comment by cmfcmf** [comment]
> The header/footer on the page for the individual devices seems to be huge! There is only about a third of the page for content. Would it make sense to change this to fixed layout that always displays the full pinout information?

You're right. I initially restricted the height of the pinout diagrams so that a potential detailed description of the µC would not be too far down the page. I opened #7 to instead move potential descriptions above the pinout diagrams and removed the height restriction on them.

> He is actually named "Dave" :)

:see_no_evil:

**Pull request merged**